ui/web-v2/apps/admin/src/modules/features.ts

```diff
@@ -288,6 +288,7 @@ export const updateFeatureTargeting = createAsyncThunk<
   request.setId(params.id);
   request.setComment(params.comment);
   request.setCommandsList(params.commands);
+  request.setFrom(UpdateFeatureTargetingRequest.From.USER);
```

Comment:
All requests from the console and e2e test must have set the `from` field as `USER`.